source/client.c

```diff
@@ -159,13 +176,16 @@ static void s_mqtt_client_init(
         return;
     }
 
+    AWS_LOGF_DEBUG(AWS_LS_MQTT_CLIENT, "Connection successfully opened, sending CONNECT packet");
+
     /* Reset the current timeout timer */
     connection->reconnect_timeouts.current = connection->reconnect_timeouts.min;
 
     /* Create the slot and handler */
     connection->slot = aws_channel_slot_new(channel);
 
     if (!connection->slot) {
+        AWS_LOGF_ERROR(AWS_LS_MQTT_CLIENT, "Failed to create new slot, something has gone horribly wrong");
```

Comment:
is the handle_error: block below worth adding a log message in?

Reply:
Nah, I'll just add logs before each error case